### DISCUSSION TITLE
Libsql support?

### DISCUSSION THREAD

**Discussion by wovalle** [beekeeper-studio/beekeeper-studio#1595] [Ideas]
First of all, thanks for this amazing product! 

I'm wondering if there is any plans to support [libsql](https://libsql.org) as an storage/db engine in the near future? libsql supports the postgres protocol but there are providers (like [turso](http://turso.tech), I'm not related to them in any way) that do not provide support for it so a native client makes sense.

If this is something that you want to pursue I could make a PR adding support for it. Given that this projects uses [knex internally](https://github.com/beekeeper-studio/beekeeper-studio/blob/b02f3f92ce742ef796e29407b794faaab9e4e042/apps/studio/src/lib/db/clients/mysql.js#L5) this shouldn't be a big thing.

There are multiple ways in which we could approach this, we could just augment the sqlite provider/screens to accept an `http://` of `libsql://` protocols apart from `file:` and use [the drop-in replacement](https://github.com/libsql/libsql-node-sqlite3) for [better-sqlite3](https://github.com/beekeeper-studio/beekeeper-studio/blob/master/apps/studio/package.json#L30) or add it as a new db engine altogether (with its own screen).

**Comment by b0o** [comment]
Looking for this too. @wovalle Do you know of any graphical DBMS tools do support libsql?
